docs/api.md

```diff
@@ -1118,19 +1118,19 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
 
 _(introduced: version 2022.3)_
 
-Dataset is a collection of entities. Dataset schema is created using form XML, you can see [ODK XForm](https://getodk.github.io/xforms-spec) specification for defining dataset in form XML. 
+A Dataset is a collection of Entities. A Dataset's schema is defined by the Forms that create Entities in that Dataset. See the [ODK XForms specification](https://getodk.github.io/xforms-spec) for defining Datasets in Form definitions.
```

Comment:
Should we add pointer to XLSForms as well? since we have pyxform updated for the datasets/entities feature

Reply:
In the API docs we usually focus on the XForms spec since that's what Central directly interacts with. I think we may want to consider shifting that over time but for now it seems most consistent with what we have to focus on the XForms spec.